src/System.Net.WebHeaderCollection/tests/WebHeaderCollectionTest.cs

```diff
@@ -38,6 +38,17 @@ public void HttpRequestHeader_Add_Success()
             Assert.Equal("Connection", w.AllKeys[0]);
         }
 
+        [Fact]
```

Comment:
The tests should instead be added to WebHeaderCollectionTest.netstandard17.cs; I expect that's the reason for the test failures.